hack/install_kubectl_plugin.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -xe
+
+REPO='https://github.com/substratusai/substratus'
+LATEST_RELEASE=$(curl ${REPO}/releases -s |
+  grep 'Link--primary' |
+  head -n1 |
+  perl -n -e '/v([0-9]+\.[0-9]+\.[0-9]+(-[a-zA-Z0-9]+)?)/ && print $&')
```

Comment:
A filthy hack because the github api doesn't seem to work unauthenticated even if chatgpt insists it does.